vue3_ts/src/components/layouts/VeevalidateYupHeader.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <nav class="mt-10">
+    <ul class="flex justify-center">
+      <router-link
+        v-for="({ link, label }, index) in links"
+        :key="index"
+        :to="{ name: link }"
+        class="mx-5 px-10 py-5 text-sm text-white bg-grey-500 hover:bg-grey-700 rounded"
+      >
```

Comment:
place it at one line with the tag closing caret

Reply:
@v-romanyuk Not sure that one line is acceptable with Vue styleguide and as for me it's less readable than current style. We are using line this in our project.

```
      <router-link
        v-for="({ link, label }, index) in links"
        :key="index"
        :to="{ name: link }"
        class="mx-5 px-10 py-5 text-sm text-white bg-grey-500 hover:bg-grey-700 rounded"
      > {{ label }} </router-link>
```

Code like above feels for me as not correct html markup and not easy to read.

Reply:
@v-romanyuk @m-tarabtsov: found this in styleguide so I think we should keep it as it was originally:
https://vuejs.org/v2/style-guide/#Keyed-v-for-essential